docs/en/sql-reference/functions/other-functions.md

```diff
@@ -3688,3 +3720,118 @@ Result:
 ```response
 {'version':'1','serial_number':'2D9071D64530052D48308473922C7ADAFA85D6C5','signature_algo':'sha256WithRSAEncryption','issuer':'/CN=marsnet.local CA','not_before':'May  7 17:01:21 2024 GMT','not_after':'May  7 17:01:21 2025 GMT','subject':'/CN=chnode1','pkey_algo':'rsaEncryption'}
 ```
+
+## lowCardinalityIndices
+
+For each row in the current [block](../../development/architecture.md/#block-block), returns the index of the value in the dictionary of unique values for columns of [LowCardinality](../data-types/lowcardinality.md) type. 
+
+:::note
+The first unique value encountered in the block is enumerated from 1.
+:::
+
+**Syntax**
+
+```sql
+lowCardinalityIndices(col)
+```
+
+**Arguments**
+
+- `col` — a low cardinality column. [LowCardinality](../data-types/lowcardinality.md).
+
+**Returned value**
+
+- returns the index of the value in the dictionary of unique values, for each row in the current block. [UInt64](../data-types/int-uint.md).
+
+**Example**
+
+Query:
+
+```sql
+DROP TABLE IF EXISTS test;
+CREATE TABLE test (s LowCardinality(String)) ENGINE = Memory;
+
+-- insert two blocks of data:
+
+INSERT INTO test VALUES ('one'),('two'),('one'),('one'),('two');
+INSERT INTO test VALUES ('three'),('two'),('one'),('two'),('two'),('three');
```

Comment:
Cosmetic: Please add spaces between the inserted values: `('one'), ('two'), [...]`

Reply:
I think example values which are stringified numbers ('one', 'two', ...) lead to confusion. We should make this more dumb and insert (in this order):
- into the 1st part: `ab`, `cd`, `ab`, `ab`, `df`.
- into the 2nd part: `ef`, `cd`, `ab`, `cd`, `ef`.